utils/ValidateReadmeTable.py

```diff
@@ -0,0 +1,62 @@
+import os
+import re
+import sys
+from typing import Final
+
+CHECK_ID_PATTERN: Final[str] = r"(PWR\d+|PWD\d+|RMK\d+)"
+CHECKS_DIR: Final[str] = "Checks/"
+README_FILE: Final[str] = "README.md"
+TABLE_MARKER: Final[str] = "|"
+
+
+def extractTableFromReadme(readmeFile: str) -> list[str]:
+    with open(readmeFile, "r") as file:
+        lines = file.readlines()
+
+    tableLines = [
+        line for line in lines if line.strip() and line.startswith(TABLE_MARKER)
+    ]
+
+    return tableLines
+
+
+def extractIdsFromTable(tableLines: list[str]) -> set[str]:
+    return {
+        match.group(1)
+        for line in tableLines
+        if (match := re.search(CHECK_ID_PATTERN, line))
+    }
```

Comment:
And merged with the function above (not sure about the indentation though):

```suggestion
def extractIdsFromTable(readmeFile: str) -> set[str]:
    with open(readmeFile, "r") as file:
        return  {
            match.group(1)
            for line in file.readlines()
            if line.startswith(TABLE_MARKER) and
            (match := re.search(CHECK_ID_PATTERN, line))
        }
```